electron/renderer/components/export-to-language-standalone.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import ExportToLanguagePlugin, { activate } from 'plugin';
+import PropTypes from 'prop-types';
+
+class ExportToLanguageStandalone extends Component {
+  static displayName = 'ExportToLanguageStandaloneComponent';
+  static propTypes = {
+    appRegistry: PropTypes.object.isRequired
+  };
+  
+  handleChange(event) {
```

Comment:
you probably already know this, but if you write this function like this you don't have to bind it on line 31 \o/. up to you tho!!
```js
handleChange = (event) => {
```